notification/ntfnhandlers.go

```diff
@@ -234,6 +236,16 @@ func MakeNodeNtfnHandlers() (*rpcclient.NotificationHandlers, *collectionQueue)
 				log.Warn("expNewTxChan buffer full!")
 			}
 
+			select {
+			case NtfnChans.InsightNewTxChan <- &insight.NewTx{
+				Time:  time.Now().Unix(),
+				Hex:   txDetails.Hex,
+				Vouts: txDetails.Vout,
+			}:
+			default:
+				log.Warn("InsightNewTxChan buffer full!")
```

Comment:
This is going to be very noising (and misleading) when running in lite mode, when `NtfnChans.InsightNewTxChan` is `nil`.